internal/mode/static/manager.go

```diff
@@ -162,6 +166,33 @@ func StartManager(cfg config.Config) error {
 		return fmt.Errorf("cannot register event loop: %w", err)
 	}
 
+	leaderElectorLogger := cfg.Logger.WithName("leaderElector")
+
+	if cfg.LeaderElection.Enabled {
+		leaderElector, err := newLeaderElectorRunnable(leaderElectorRunnableConfig{
+			kubeConfig: clusterCfg,
+			recorder:   recorder,
+			onStartedLeading: func(ctx context.Context) {
+				leaderElectorLogger.Info("Started leading")
```

Comment:
nit: Is there any reason why `onStartedLeading` has the logging before calling `statusUpdater.Enable()` while `onStoppedLeading` has it after? If not, could we make those consistent with each other?